doc/classes/BakedLightmap.xml

```diff
@@ -18,12 +18,14 @@
 			<argument index="1" name="create_visual_debug" type="bool" default="false">
 			</argument>
 			<description>
+				Bakes the lightmaps within the currently-editted scene.
```

Comment:
Sorry to bother again, but there's still the duplicated "t" from before :slightly_smiling_face:

```suggestion
				Bakes the lightmaps within the currently-edited scene.
```

Reply:
Oops, good catch! This should be fixed now :)